src/authenticated-app.js

```diff
@@ -31,7 +34,11 @@ function ErrorFallback({ error }) {
 }
 
 function AuthenticatedApp() {
-  const { user, logout } = useAuth();
+  const { user } = useSelector(selectUser);
+  const dispatch = useDispatch();
+  const handleLogout = () => dispatch(logout());
```

Comment:
We need to reset book items and list items (new actions) when the user logs out. If not, data will be cached if new user logins.